types/mixpanel-browser/index.d.ts

```diff
@@ -159,12 +193,28 @@ export function opt_out_tracking(options?: Partial<OutTrackingOptions>): void;
 export function push(item: PushItem): void;
 export function register(props: Dict, days?: number): void;
 export function register_once(props: Dict, default_value?: any, days?: number): void;
+export function remove_group(
+    group_key: string,
+    group_ids: string | string[] | number | number[],
+    callback?: Callback,
+): void;
 export function reset(): void;
 export function set_config(config: Partial<Config>): void;
+export function set_group(
+    group_key: string,
+    group_ids: string | string[] | number | number[],
+    callback?: Callback,
+): void;
 export function time_event(event_name: string): void;
-export function track(event_name: string, properties?: Dict, optionsOrCallback?: RequestOptions | Callback, callback?: Callback): void;
+export function track(
+    event_name: string,
+    properties?: Dict,
+    optionsOrCallback?: RequestOptions | Callback,
+    callback?: Callback,
+): void;
 export function track_forms(query: Query, event_name: string, properties?: Dict | (() => void)): void;
 export function track_links(query: Query, event_name: string, properties?: Dict | (() => void)): void;
+export function track_with_groups(event_name: string, properties: Dict, groups: Dict, callback?: Callback): void;
```

Comment:
This seems to be missing from the Mixpanel interface above.

Reply:
Fixed